lib/composables/useKShow/__tests__/index.spec.js

```diff
@@ -0,0 +1,42 @@
+import useKShow from '../index';
+
+const { show } = useKShow();
+
+describe('useKShow', () => {
+  beforeAll(() => {
+    // https://github.com/jestjs/jest/issues/11607
+    jest.useFakeTimers('legacy');
+  });
+  afterAll(() => {
+    jest.useRealTimers();
+  });
```

Comment:
Love how simple this setup is for the time management

Reply:
Yes, I was quite surprised :)